tests/docker-compose.yaml

```diff
@@ -2,23 +2,52 @@ services:
   minio:
     image: docker.io/minio/minio
     command: server /data
-    network_mode: "host"
+    ports:
+      - "9000:9000"
     volumes:
       - ./data:/data
-  databend:
-    image: docker.io/datafuselabs/databend:nightly
+  meta:
+    image: docker.io/datafuselabs/databend-meta:nightly
+    volumes:
+      - ./config/databend-meta-node-1.toml:/conf.toml:ro
+      - ./check_port.pl:/check_port.pl:ro
+    command: -c /conf.toml
+    ports:
+      - "28101:28101"
+    healthcheck:
+      test: [ "CMD-SHELL", "perl /check_port.pl 28101" ]
+      interval: 2s
+      retries: 10
+      start_period: 2s
+      timeout: 1s
+  query:
+    image: docker.io/datafuselabs/databend-query:nightly
+    volumes:
+      - ./config/databend-query-node-1.toml:/conf.toml:ro
+      - ./check_port.pl:/check_port.pl:ro
+    command: -c /conf.toml
     environment:
-      - QUERY_STORAGE_TYPE=s3
       - QUERY_DATABEND_ENTERPRISE_LICENSE
-      - AWS_S3_ENDPOINT=http://localhost:9000
-      - AWS_ACCESS_KEY_ID=minioadmin
-      - AWS_SECRET_ACCESS_KEY=minioadmin
-    network_mode: "host"
+    ports:
+      - "8900-8902:8900" # flight sql handler
+      # - "8001-8003:8000" # http handler
     depends_on:
-      - minio
+      minio:
+        condition: service_started
+      meta:
+        condition: service_healthy
     healthcheck:
-      test: "curl -f localhost:8080/v1/health || exit 1"
+      test: [ "CMD-SHELL", "perl /check_port.pl 8000" ]
```

Comment:
why changing this?

Reply:
thanks, no need for query. meta image do not contain curl

Reply:
you could use `/databend-metactl status` for meta

Reply:
done, thank you!